qemu/tests/rng_bat.py

```diff
@@ -61,6 +66,10 @@ def is_dev_used_by_qemu(dev_file, vm_pid):
         session = utils_test.qemu.windrv_check_running_verifier(session, vm,
                                                                 test, driver_name,
                                                                 timeout)
+        if not system.file_exists(session,
+                                  "c:\\random_%PROCESSOR_ARCHITECTURE%.exe"):
```

Comment:
I think you miss this one, `if not system.file_exists(session, rng_dst)`

Reply:
Yes, you are right, updated, thanks